src/imex.rs

```diff
@@ -269,24 +269,47 @@ async fn import_backup(
         context.get_dbfile().display()
     );
 
+    import_backup_stream(context, backup_file, file_size, passphrase).await?;
+    Ok(())
+}
+
+/// Imports backup by reading a tar file from a stream.
+///
+/// `file_size` is ideally the size of the tar file,
+/// but can be an estimate such as the sum of all
```

Comment:
Then maybe better if it is the sum of all files' sizes, if that's what it actually is? Then the progress can be calculated more accurately (without error accumulation)

Reply:
Then when streaming a backup from another device it will hang at 99% for some time because the metadata size isn't accounted. This is what users generally don't like as it seems like the process is stuck. Maybe just multiply the progress by e.g. 0.95 so it will be a little slower, but w/o stucking at 99%?